docs/examples.md

```diff
@@ -41,6 +41,51 @@ pipx run --spec https://github.com/psf/black/archive/18.9b0.zip black --help
 pipx run https://gist.githubusercontent.com/cs01/fa721a17a326e551ede048c5088f9e0f/raw/6bdfbb6e9c1132b1c38fdd2f195d4a24c540c324/pipx-demo.py
 ```
 
+You can run local files, or scripts hosted on the internet, and you can run them with arguments:
+
+```
+pipx run test.py
+pipx run test.py 1 2 3
+pipx run https://example.com/test.py
+pipx run https://example.com/test.py 1 2 3
+```
+
+A simple filename is ambiguous - it could be a file, or a package on PyPI. It
+will be treated as a filename if the file exists, or as a package if not. To
+force interpretation as a local path, use `--path`, and to force interpretation
+as a package name, use `--spec` (with the PyPI name of the package).
+
+```
+pipx run myscript.py # Local file, if myscript.py exists
+pipx run doesnotexist.py # Package, because doesnotexist.py is not a local file
+pipx run --path test.py # Always a local file
+pipx run --spec test-py test.py # Always a package on PyPI
+```
+
+You can also run scripts that have dependencies:
+
+If you have a script `test.py` that needs a 3rd party library like requests:
+
+```
+# test.py
+
+# Requirements:
+# requests
+
```

Comment:
Might want to explain briefly what would be necessary if I want to add comments after the `# Requirements:` line.

Reply:
Sure. Would just adding an example be enough?

```
# Requirements:
# requests
#
# The list of requirements is terminated by a blank line or an empty comment line.
```